cluster/examples/kubernetes/ceph/pool-mirrored.yaml

```diff
@@ -0,0 +1,20 @@
+#################################################################################################################
+# Create a mirroring enabled Ceph pool. Only a single OSD is required.
+#  kubectl create -f pool-mirrored.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: mirroredpool
+  namespace: rook-ceph
+spec:
+  replicated:
+    size: 1
+  mirroring:
+    enabled: true
+    mode: image
+    # schedule(s) of snapshot
+    snapshotSchedules:
```

Comment:
The `VolumeReplicationClass` passes snap sched already, do we need that?

Reply:
@Yuggupta27 Can this be removed as @leseb suggested?